src/ngraph/frontend/onnx_import/op/conv.hpp

```diff
@@ -0,0 +1,162 @@
+/*******************************************************************************
+ * Copyright 2018 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+#pragma once
+
+#include <cstddef>
+#include <memory>
+#include <vector>
+
+#include "ngraph/frontend/onnx_import/exceptions.hpp"
+#include "ngraph/frontend/onnx_import/node.hpp"
+#include "ngraph/frontend/onnx_import/util/conv_pool.hpp"
+
+#include "ngraph/coordinate_diff.hpp"
+#include "ngraph/node.hpp"
+#include "ngraph/node_vector.hpp"
+#include "ngraph/op/add.hpp"
+#include "ngraph/op/broadcast.hpp"
+#include "ngraph/op/concat.hpp"
+#include "ngraph/op/convolution.hpp"
+#include "ngraph/op/op.hpp"
+#include "ngraph/op/slice.hpp"
+#include "ngraph/strides.hpp"
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        namespace op
+        {
+            namespace detail
+            {
+                inline std::shared_ptr<ngraph::op::Op>
+                    make_ng_convolution(const std::shared_ptr<ngraph::Node>& data,
+                                        const std::shared_ptr<ngraph::Node>& filters,
+                                        const ngraph::Strides& strides,
+                                        const ngraph::Strides& dilations,
+                                        const ngraph::CoordinateDiff& padding_below,
+                                        const ngraph::CoordinateDiff& padding_above,
+                                        int groups)
+                {
+                    if (groups != 1)

```

Comment:
Implementation should be moved to .cpp file. Please review nGraph coding conventions.